MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/camera/ZoomToShowClusterLeavesActivity.java

```diff
@@ -0,0 +1,270 @@
+
+    // This contains the MapView in XML and needs to be called after the access token is configured.
+    setContentView(R.layout.activity_zoom_to_show_cluster_leaves);
+
+    mapView = findViewById(R.id.mapView);
+    mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(new OnMapReadyCallback() {
+      @Override
+      public void onMapReady(@NonNull MapboxMap map) {
+        map.setStyle(Style.DARK, new Style.OnStyleLoaded() {
+          @Override
+          public void onStyleLoaded(@NonNull Style style) {
+
+            mapboxMap = map;
+
+            // Disable any type of fading transition when icons collide on the map. This enhances the visual
+            // look of the data clustering together and breaking apart.
+            style.setTransition(new TransitionOptions(0, 0, false));
+
+            mapboxMap.animateCamera(CameraUpdateFactory.newLatLngZoom(new LatLng(12.099, -79.045), 3));
+
+            addClusteredGeoJsonSource(style);
+
+            style.addImage(
+              "icon-id",
+              BitmapUtils.getBitmapFromDrawable(getResources().getDrawable(R.drawable.single_quake_icon)));
+
+            mapboxMap.addOnMapClickListener(ZoomToShowClusterLeavesActivity.this);
+
+            Toast.makeText(ZoomToShowClusterLeavesActivity.this,
+                R.string.tap_on_cluster_circle_instruction, Toast.LENGTH_SHORT).show();
+          }
+        });
+      }
+    });
+  }
+
+  @Override
+  public boolean onMapClick(@NonNull LatLng point) {
+    PointF pointf = mapboxMap.getProjection().toScreenLocation(point);
+    RectF rectF = new RectF(pointf.x - 10, pointf.y - 10, pointf.x + 10, pointf.y + 10);
+    List<Feature> mapClickFeatureList = mapboxMap.queryRenderedFeatures(rectF, queryLayerIds);
+    if (mapClickFeatureList.size() > 0) {
+      FeatureCollection clusterLeavesFeatureCollection = geoJsonSource.getClusterLeaves(mapClickFeatureList.get(0),
+        8000, 0);
+      moveCameraToLeavesBounds(clusterLeavesFeatureCollection);
+    }
+
+    return true;
+  }
+
+  private void moveCameraToLeavesBounds(FeatureCollection featureCollectionToInspect) {
+    List<LatLng> latLngList = new ArrayList<>();
+    if (featureCollectionToInspect.features() != null) {
+      for (Feature singleClusterFeature : featureCollectionToInspect.features()) {
+        Point clusterPoint = (Point) singleClusterFeature.geometry();
+        if (clusterPoint != null) {
+          latLngList.add(new LatLng(clusterPoint.latitude(), clusterPoint.longitude()));
+        }
+      }
+      if (latLngList.size() > 1) {
+        LatLngBounds latLngBounds = new LatLngBounds.Builder()
+          .includes(latLngList)
+          .build();
+        mapboxMap.easeCamera(CameraUpdateFactory.newLatLngBounds(latLngBounds, 230),
+          1300);
+      }
+    }
+  }
+
+  private void addClusteredGeoJsonSource(@NonNull Style loadedMapStyle) {
+
+    // Add a new source from the GeoJSON data and set the 'cluster' option to true.
```

Comment:
Small suggestion, since double quotes are being used elsewhere.
```suggestion
    // Add a new source from the GeoJSON data and set the "cluster" option to true.
```